hikari/api/rest.py

```diff
@@ -5047,6 +5047,107 @@ async def edit_welcome_screen(
             If an internal error occurs on Discord while handling the request.
         """
 
+    @abc.abstractmethod
+    async def fetch_guild_membership_gate(
+        self, guild: snowflakes.SnowflakeishOr[guilds.PartialGuild]
+    ) -> guilds.MembershipGate:
+        """Fetch a guild's membership verification gate screen.
+
+        Parameters
+        ----------
+        guild : hikari.snowflakes.SnowflakeishOr[hikari.guilds.PartialGuild]
+            Object or ID of the guild to fetch the member verification screen for
+
+        Returns
+        -------
+        hikari.guilds.MembershipGate
+            The requested membership verification gate screen.
+
+        Raises
+        ------
+        hikari.errors.ForbiddenError
+            If you are not part of the guild or if you are missing the
+            "MANAGE_GUILD" permission or if the guild does not have access to the
+            feature (is not a community guild).
+        hikari.errors.NotFoundError
+            If the guild is not found.
+        hikari.errors.UnauthorizedError
+            If you are unauthorized to make the request (invalid/missing token).
+        hikari.errors.RateLimitTooLongError
+            Raised in the event that a rate limit occurs that is
+            longer than `max_rate_limit` when making a request.
+        hikari.errors.RateLimitedError
+            Usually, Hikari will handle and retry on hitting
```

Comment:
is hikari canonically lower or uppercase? (the readme has lowercase hikari)

Reply:
It should always be upper case in docs and comments since it's a name (same would apply to stuff like Discord but we're not as consistent with that one lol)